react/__mocks__/testUtils.ts

```diff
@@ -0,0 +1,13 @@
+export function findCSSHandles(
+  container: HTMLElement,
+  handles: readonly string[]
+) {
+  const foundNodes = handles
+    .map(handle => {
+      const foundNodesInner = container.getElementsByClassName(handle)
+      return foundNodesInner.length > 0 ? handle : ''
+    })
+    .filter(result => result !== '')
+
+  return foundNodes
+}
```

Comment:
Uhh nice

Reply:
It was totally my idea, is nice that @victorhmp liked too